packages/manager/modules/core/src/redirection/redirection.constants.js

```diff
@@ -26,6 +26,7 @@ const URI = {
   paymentMethod: '#/billing/payment/method',
   services: '#/billing/autoRenew',
   support: '#/support/tickets',
+  newSupportTicket: '#/support/tickets/new',
```

Comment:
If I'm correct something similar was added to `develop` for the account sidebar, it might be worth to check to avoid duplicating this redirection

Reply:
We are now using `createTicket` which is already available. Thanks for pointing this out :)